portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,53 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+body {
+    background-color: white;
+    color: black;
+}
+
+h1 {
+    text-align: center;
+    background-color:lightgray;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+footer {
+    text-align: center;
+    background-color: black;
+    color: white;
+}
+
+.box-layout {
+    display: inline-block;
+    width: 400px;
+    height: 100px;
+    margin: 1px;
+    border: 1px;
+}
+
+h3 {
+    text-align: center;
+}
+
+.button {
+    border: none;
+    text-align: center;
+    cursor: pointer;
+    padding: 5px 10px;
+    margin: 4px 2px;
+    border-radius: 10px;
+}
+
+.buttonBlue {background-color: #0e76a8; color:white;}
+.buttonGray {background-color: #e8e8e8; color:black;}
```

Comment:
Tip: You can use a predefined colour for the `background-color` attribute too.
[List of predefined colours](https://www.w3schools.com/colors/colors_names.asp)

Reply:
Thanks for the tip! I had a look, but the shades didn't seem quite right.